salt/modules/rsync.py

```diff
@@ -68,7 +68,8 @@ def rsync(src,
           exclude=None,
           excludefrom=None,
           dryrun=False,
-          rsh=None):
+          rsh=None,
+          additional_opts=None):
```

Comment:
Could you please add docs that explain how this operates? It would be nice if the rest were documented as well, but as this sits right now, I'm not sure how a user would understand how to use it or the fact that `additional_opts` needs to be a list.